samples/ballerina-by-example/examples.list

```diff
@@ -1 +1,2 @@
 ddfd
+While
```

Comment:
Remove the changes in this file since it will conflict whenever PR is merge with the changes to this file.